### ISSUE TITLE
Loading the model

### ISSUE THREAD

**Issue by Mukish45** [usefulsensors/openai-whisper#20]
Hello @nyadla-sys. You did a great job on making the whisper model working on Android. I'm also trying with the same, but struggling to load and run the inference using C/C++ library. Could you please share any references on setting up those C/C++ libraries.

**Comment by nyadla-sys** [comment]
I followed the steps from here to link the required libs.
https://github.com/cuongvng/TF-Lite-Cpp-API-for-Android

You can compile and run the attached zip file for your Android phone.
https://github.com/usefulsensors/openai-whisper/blob/main/android_app/android_whisper_app.zip

**Comment by Mukish45** [comment]
Is there any other alternative ways to run whisper model on android without using C++ libraries?

**Comment by nyadla-sys** [comment]
@Mukish45 Definitely, the app can be run using Java APIs, but I am not an expert in writing Android applications

**Comment by nyadla-sys** [comment]
Yes it is possible you need to use to different [tflite model ](https://github.com/usefulsensors/openai-whisper/blob/main/models/whisper-base.tflite)for translate in Android App


refer the below comments from this thread https://github.com/usefulsensors/openai-whisper/issues/15
This also might be of interest to you @nyadla-sys The base model do translation to english where as the tiny and small models just returned the language detected.

mycroft@OpenVoiceOS-e3830c:~/whisper $ minimal models/whisper-tiny.tflite de_speech_thorsten_sample03_8s.wav

n_vocab:50257

mel.n_len3000

mel.n_mel:80
INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
Inference time 7 seconds

[_extra_token_50258][_extra_token_50261][_extra_token_50359][_BEG_] Für mich sind alle Menschen gleich unabhängig von Geschlecht, sexuelle Orientierung, Religion, Hautfarbe oder Geo-Kordinaten der Geburt.[_SOT_]

mycroft@OpenVoiceOS-e3830c:~/whisper $ minimal models/whisper-base.tflite de_speech_thorsten_sample03_8s.wav

n_vocab:50257

mel.n_len3000

mel.n_mel:80
INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
Inference time 12 seconds

[_extra_token_50258][_extra_token_50261][_extra_token_50358][_BEG_] For me, all people are equally independent

**Comment by nyadla-sys** [comment]
Actually some one is trying to implement multilanguage(transcribe and translate) support on Android APP using TFLite models

Refer the below code for more details..

https://github.com/ipsilondev/whisper-cordova/blob/main/android/cpp/native-lib.cpp

**Comment by Mukish45** [comment]
Thank you @nyadla-sys for your quick response also for sharing reference.
